pkg/apis/tenancy/v1alpha1/types.go

```diff
@@ -84,6 +84,7 @@ type ClusterWorkspaceSpec struct {
 	//
 	// +optional
 	// +kubebuilder:default:="Universal"
+	// +kubebuilder:validation:Pattern=`^[A-Z][a-zA-Z0-9]*[a-zA-Z0-9]$`
```

Comment:
Is this: at least 2 chars long, first must be capital letter, all subsequent are lowercase/uppercase/digit?

Reply:
CaML12Case321. Yes, 2 chars min length (we have that in a couple of places).

Reply:
Can simplify regex. Use a single + instead of * then repeating the group

Reply:
updated